### PR TITLE
Ordered restarts for kafka/zk and serial restarts for other components

### PR DESCRIPTION
**Pull request by domenicbove**
# Description

Kafka and ZK should have their restarts ordered
Handlers should be removed from the variables and handlers role (they only caused unordered handler invocations), and therefore the name of the role should change

Rest of the components should have serial restarts (unordered though)

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Ran a few installs with 1-3 brokers,zk nodes
Ran the upgrade kafka broker playbook
Ran serial component restart with multiple hosts in each group

# Checklist:

- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: roles/confluent.kafka_broker/tasks/ordered_restart.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+- name: Get ActiveControllerCount
+  uri:
+    url: "{{ kafka_broker_jolokia_active_controller_url }}"
+    validate_certs: false
+    return_content: true
+    status_code: 200
+  register: active_controller_count_query
+  check_mode: false
+
+- set_fact:
```
Comment (domenicbove) [inline roles/confluent.kafka_broker/tasks/ordered_restart.yml:11]: is this set fact necessary? just move to the conditional?

File: roles/confluent.test/molecule/plain-rhel/molecule.yml [hunk 2]
```diff
@@ -26,6 +26,30 @@ platforms:
     privileged: true
     networks:
       - name: confluent
+  # - name: kafka-broker2
```
Comment (domenicbove) [inline roles/confluent.test/molecule/plain-rhel/molecule.yml:29]: revert this file to original

File: roles/confluent.zookeeper/tasks/ordered_restart.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+# All zookeeper hosts iterate over the list of zookeeper hosts, each seeing the list in the same order
+# At one given loop entry, only one host will have its inventory_hostname match  the entry in the loop
+# restart.yml tasks will only run for that host ON that host
+- name: Restart Followers
+  include: restart.yml
+  delegate_to: "{{ item }}"
+  loop: "{{ groups['zookeeper'] }}"
+  when:
+    - "hostvars[item].inventory_hostname == inventory_hostname"
+    # Edge case that zk is in standalone mode (ie only on node)
```
Comment (domenicbove) [inline roles/confluent.zookeeper/tasks/ordered_restart.yml:19]: ie only one node

File: roles/confluent.test/molecule/plain-rhel/molecule.yml [hunk 4]
```diff
@@ -26,6 +26,30 @@ platforms:
     privileged: true
     networks:
       - name: confluent
+  - name: kafka-broker2
```
Comment (domenicbove) [inline roles/confluent.test/molecule/plain-rhel/molecule.yml:29]: remove these new lines

File: upgrade_kafka_broker_log_format.yml [hunk 5]
```diff
@@ -22,7 +22,7 @@
     kafka_broker_upgrade_end_version: "5.5"
   tasks:
     - import_role:
-        name: confluent.variables_handlers
```
Comment (domenicbove) [inline upgrade_kafka_broker_log_format.yml:25]: this task file is missing the new handlers!
